apps/earn-protocol/app/server-handlers/sdk/get-vaults-list.ts

```
@@ -1,27 +1,34 @@
+      }),
+    )
 
-  // flatten the list
-  const vaultsFlatList = vaultsListByNetwork.reduce<(typeof vaultsListByNetwork)[number]['vaults']>(
-    (acc, { vaults }) => [...acc, ...vaults],
-    [],
-  )
+    // flatten the list
+    const vaultsFlatList = vaultsListByNetwork.reduce<
+      (typeof vaultsListByNetwork)[number]['vaults']
+    >((acc, { vaults }) => [...acc, ...vaults], [])
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)